addons/point_of_sale/models/pos_order.py

```diff
@@ -1136,6 +1136,10 @@ def get_sale_details(self, date_start=False, date_stop=False, configs=False):
         """
         if not configs:
             configs = self.env['pos.config'].search([])
+        elif isinstance(configs, int):
```

Comment:
indeed, but since we change the use of the signature to allow not only recordsets, but also lists of ids for the JS to be able to use the function, I think getting it explicitly ready to accept integers is not overkill

Reply:
Ok for me if you think its best to keep it. :+1: